apps/console/src/protected-app.tsx

```diff
@@ -213,7 +213,7 @@ export const ProtectedApp: FunctionComponent<AppPropsInterface> = (): ReactEleme
                 has(idToken, "associated_tenants") ||
                 isPrivilegedUser ||
                 isOrgSwitch ||
-                isNotPlatformIdPFederatedUser
+                isNotPlatformIdPFederatedUser !== undefined && isNotPlatformIdPFederatedUser
```

Comment:
Why are we fixing this ? its seems like we are making `isNotPlatformIdPFederatedUser` undefined and checking it in this line, but the previous way also did the same right ?

Reply:
This is due to the mentioned issue in the PR. Previously, `isNotPlatformIdPFederatedUser ` was set to true by default and if experimental_platformIdp is false (in legacy mode), the `isNotPlatformIdPFederatedUser` value will always be true. Because of that, even if the user does not has associated tenants, the user was logged into the console instead of sending to create tenant page.

With this fix, since we are checking if `not undefined` and is true to send the user to console app, the above issue does not arise.

Reply:
@Achintha444